lib/api/serdes/obms.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var injector = require('../../../index').injector;
+var serializer = injector.get('Http.Services.Swagger').serializer;
+var deserializer = injector.get('Http.Services.Swagger').deserializer;
```

Comment:
@srinia6 You can get rid of `deserializer` since it is not used.  Also you could create a separate variable for serializer so that you don't to instantiate two identical serializers.

    var obmSerializer = serializer('Serializables.V1.Obm');
    module.exports = {
        getObmLib: obmSerializer,
        getObmLibById: obmSerializer
    };